schema.graphql

```diff
@@ -67,3 +48,35 @@ type UniquesTransfer @entity {
   timestamp: Float @index
   success: Boolean
 }
+
+type Pot @entity {
+  id: ID!
+  potId: Float! @index 
+  owner: String! @index
+  feeQuota: BigInt!
+  reserveQuota: BigInt!
+  sponsorshipType: String!
+  sponsoredAccounts: [PotBalance] @derivedFrom(field: "pot")
+  createdAt: Float @index
+  updatedAt: Float @index
+
+}
+
+type PotBalance @entity @compositeIndexes(fields: [["pot", "account"]]) {
```

Comment:
In [substrate docs](https://academy.subquery.network/build/graphql.html#composite-index) they mention the composite index can be used over standard scalar types while I see in their own example they have used for the account type too which. Do you know what they have meant by the standard scalar types?

Reply:
@aliXsed "Scalar types" are the same as "Primitive values", the difference is the context, while Scalar types is common use in Database, Primitive is used in Programming